src/main/java/persistence/sql/ddl/dialect/H2ColumnType.java

```diff
@@ -0,0 +1,39 @@
+package persistence.sql.ddl.dialect;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import persistence.sql.ddl.exception.NotSupportedTypeException;
+
+/**
+ * @http://www.h2database.com/html/datatypes.html
+ */
+public class H2ColumnType implements ColumnType {
+
+    private final Map<Class<?>, String> typeMap = new HashMap<>();
+
+    private static final HashMap.Entry<Class<?>, String> longType = Map.entry(Long.class, "bigint");
+    private static final HashMap.Entry<Class<?>, String> stringType = Map.entry(String.class, "varchar");
+    private static final HashMap.Entry<Class<?>, String> integerType = Map.entry(Integer.class, "integer");
+
+    public H2ColumnType() {
+        typeMap.put(longType.getKey(), longType.getValue());
```

Comment:
아래처럼 작성 되어도 괜찮을 것 같은데 별도로 HashMap.Entry를 선언해주신 이유가 궁금합니다 :)

```suggestion
        typeMap.put(Long.class, "bigint");
```

Reply:
초기작성시에는 제안해주신 것처럼 작성했었는데 연관성 있는 타입을 하나로 묶어 실수를 줄이고 싶다는 의도로 변경했습니다!